src/argilla/client/sdk/v1/datasets/models.py

```diff
@@ -131,8 +131,7 @@ class FeedbackRecordsMetricsModel(BaseModel):
 class FeedbackVectorSettingsModel(BaseModel):
     id: UUID
     name: str
-    # TODO: Uncomment when is supported
-    # title: Optional[str] = None
+    title: Optional[str] = None
```

Comment:
Since the `title` is not optional in the server part, I would say to align both schemas. A different thing is to set the default title as the vector name.